tgui/packages/tgui/interfaces/OverwatchConsole.js

```diff
@@ -0,0 +1,860 @@
+  );
+
+  return (
+    <Section
+      title={currentCategory === 1 ? 'Supply Drop' : 'Orbital Bombardment'}>
+      <Stack>
+        <Stack.Item grow>
+          <Button
+            fluid
+            selected={currentCategory === 1}
+            textColor={currentCategory === 1 ? '#ffffff' : '#161613'}
+            icon="box-open"
+            onClick={() => setCategory(1)}
+            content="Supply Drop"
+          />
+        </Stack.Item>
+        <Stack.Item grow>
+          <Button
+            fluid
+            selected={currentCategory === 2}
+            icon="burst"
+            textColor={currentCategory === 2 ? '#ffffff' : '#161613'}
+            onClick={() => setCategory(2)}
+            content="Orbital Bombardment"
+          />
+        </Stack.Item>
+      </Stack>
+      <Divider />
+      <Box>
+        {currentCategory === 1 ? <OverwatchSupplies /> : <OverwatchBomb />}
+      </Box>
+    </Section>
+  );
+};
+
+export const OverwatchEmpty = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Section flexGrow="1" fill>
+      <OverwatchId />
+      <Flex height="100%" textAlign="center">
+        <Flex.Item grow="1" align="center" colour="red">
+          <Icon name="times-circle" mb="0.5rem" size="5" colour="red" />
+          <br />
+          Authentication required.
+          <br />
+        </Flex.Item>
+      </Flex>
+    </Section>
+  );
+};
+
+export const OverwatchId = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { operator, authenticated } = data;
+  return (
+    <Section title="Authentication">
+      <Stack>
+        <Stack.Item grow>
+          <Button
+            icon="user"
+            fluid
+            content={
+              operator ? 'Operator: ' + operator : 'Unauthenticated'
+            }
+            onClick={() => act('change_operator')}
+          />
+        </Stack.Item>
+        <Stack.Item>
+          <Button
+            icon="sign-in-alt"
+            fluid
+            content={`Sign Out`}
+            disabled={!operator}
+            selected={authenticated}
+            onClick={() => act('logout')}
+          />
+        </Stack.Item>
+      </Stack>
+    </Section>
+  );
+};
+
+export const OverwatchSelect = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { squad_data, squad_list } = data;
+  return (
+    <Section
+      title="Squad Selection"
+      buttons={
+        <Dropdown
+          options={squad_list}
+          onSelected={(value) => act('pick_squad', { squadpicked: value })}
+          selected="Select"
+          noscroll
+          displayText="Select..."
+          lineHeight="1.5em"
+          height="1.8em"
+        />
+      }>
+      <Stack>
+        <Stack.Item grow>
+          <Button
+            textAlign="center"
+            icon={'user'}
+            fluid
+            content={
+              squad_data ? 'Squad: ' + squad_data.squad_name : 'Squad: None'
```

Comment:
You could use null coalescence here for something like

`'Squad: ' + squad_data?.squad_name ?? 'None'`